### ISSUE TITLE
Using q-layout yields undefined error

### ISSUE THREAD

**Issue by PGimenez** [GenieFramework/StippleUI.jl#107]
Trying to use `q-layout`either in HTML or with StippleUI throws a ` Error:  UndefVarError: `q__layout` not defined`

MWE
```julia
using GenieFramework
@genietools

@app begin
    @out var1 = 0
end

function ui()
     # StippleUI.layout(["aaa"])
     "<q-layout></q-layout>"
end
@page("/", ui)
```

**Comment by hhaensel** [comment]
Not sure, why this is. It seems that `ui()` is parsed. But even if I wrapped it in ParsedHTMLString, it was still parsed.
My way around was to manually define the route:
```julia
route("/") do 
    model = @init
    page(model, ui) |> html
end
```

**Comment by essenciary** [comment]
@PGimenez @hhaensel addressed in this commit, keep an eye on the 1.20 release
https://github.com/GenieFramework/GenieFramework.jl/commit/9e149daab4681f4e3d6bb4d7da45deaf42067cf7
